boards/telosb/board.c

```diff
@@ -1,10 +1,10 @@
+/*
+ * SPDX-FileCopyrightText: 2013 Oliver Hahm <oliver.hahm@inria.fr>
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
 /*
  * board.c - Board initiazilation for the TelosB
```

Comment:
```suggestion
 /**
 * @ingroup     boards_telosb
 *
 * @{
 *
 * @file
 * @brief       Board initialization for the TelosB
 *
 * @author      Oliver Hahm <oliver.hahm@inria.fr>
```

While we are at it, we should probably give this file a proper header. Remember to add the closing tag at the bottom too (that line is out of reach for suggestions).